### PR TITLE
Update flashloan bot based on feedback

### PR DESCRIPTION
**Pull request by dikel**

### CODE REVIEW

File: flashloan-detector/src/agent.js [hunk 1]
```diff
@@ -83,16 +86,22 @@ function provideHandleTransaction(helper, getFlashloans) {
...
 
     const totalProfit = tokensUsdProfit + nativeUsdProfit;
     const percentage = (totalProfit / totalBorrowed) * 100;
 
-    if (percentage > PERCENTAGE_THRESHOLD && totalProfit > PROFIT_THRESHOLD) {
+    console.log('Chain     :', chain);
+    console.log('TX hash   :', txEvent.hash);
+    console.log('Borrowed  :', totalBorrowed.toFixed(2));
+    console.log('Profit    :', totalProfit.toFixed(2));
+    console.log('Percentage:', percentage.toFixed(2));
```
Comment (notyouraveragedev) [inline flashloan-detector/src/agent.js:99]: Is this for logging purposes?

File: flashloan-detector/src/detectors/euler-detector.js [hunk 2]
```diff
@@ -81,12 +81,7 @@ module.exports = {
         .map((event) => event.args.amount)
         .filter((a) => withdrawn.toString().startsWith(a.toString()))[0];
 
-      // console.log(txEvent
-      //   .filterLog(requestBorrowEvent, address)
-      //   .filter((event) => {
-      //     console.log(event, account)
-      //     return event.args.account === account
-      //   }));
```
Comment (Tdrachev) [inline flashloan-detector/src/detectors/euler-detector.js:89]: Do we need this?

**Review by notyouraveragedev — CHANGES_REQUESTED** [review]

File: flashloan-detector/src/agent.js (hunk 1 above)
Comment (dikel) [reply flashloan-detector/src/agent.js:99]: Yes. In case there is a flashloan that isn't flagged in the future

**Review by notyouraveragedev — APPROVED** [review]
LGTM

**Pull request merged**